examples/LGACSend.ino

```diff
@@ -0,0 +1,258 @@
+// Copyright 2017 xpokor22
+#include <IRremoteESP8266.h>
+#include <IRsend.h>
+#include <ir_LG.h>
+
+IRsend irsend(14);  // An IR LED is controlled by GPIO pin 14 (D5)
+
+// 0 : TOWER
+// 1 : WALL
+const unsigned int AC_TYPE  = 1;
+
+// 0 : cooling
+// 1 : heating
+unsigned int AC_HEAT = 1;
+
+// 0 : off
+// 1 : on
+unsigned int AC_POWER_ON = 0;
+
+// 0 : off
+// 1 : on --> power on
+unsigned int AC_AIR_ACLEAN = 0;
+
+// temperature : 18 ~ 30
+unsigned int AC_TEMPERATURE = 24;
+
+// 0 : low
+// 1 : mid
+// 2 : high
+// if AC_TYPE = 1, 3 : change
+unsigned int AC_FLOW = 0;
+
+const int AC_FLOW_TOWER[3] = {0, 4, 6};
+const int AC_FLOW_WALL[4] = {0, 2, 4, 5};
+
+unsigned long AC_CODE_TO_SEND;
+
+void ac_send_code(unsigned long code) {
+  Serial.print("code to send : ");
+  Serial.print(code, BIN);
+  Serial.print(" : ");
+  Serial.println(code, HEX);
+
+  irsend.sendLG(code, 28);
+}
+
+void ac_activate(unsigned int temperature, unsigned int air_flow,
+		 unsigned int heat) {
+  AC_HEAT = heat;
+  unsigned int AC_MSBITS1 = 8;
+  unsigned int AC_MSBITS2 = 8;
+  unsigned int AC_MSBITS3 = 0;
+  unsigned int AC_MSBITS4 ;
+  if (AC_HEAT == 1)
+    AC_MSBITS4 = 4;  // heating
+  else
+    AC_MSBITS4 = 0;  // cooling
+  unsigned int AC_MSBITS5 =  (temperature < 15) ? 0 : temperature - 15;
+  unsigned int AC_MSBITS6 ;
+
+  if (AC_TYPE == 0)
+    AC_MSBITS6 = AC_FLOW_TOWER[air_flow];
+  else
+    AC_MSBITS6 = AC_FLOW_WALL[air_flow];
+
+  int AC_MSBITS7 = (AC_MSBITS3 + AC_MSBITS4 + AC_MSBITS5 + AC_MSBITS6) & B00001111;
+
+  AC_CODE_TO_SEND = AC_MSBITS1 << 4 ;
```

Comment:
This block of code is confusing and looks wrong. Please add some comments and/or re-write so it is clearer.